tests/functional/adapter/test_basic.py

```diff
@@ -100,6 +100,19 @@ def test_incremental(self, project):
         assert len(catalog.nodes) == 3
         assert len(catalog.sources) == 1
 
+incremental_partitionby_sql = """
+ {{ config(materialized="incremental", partition_by="id_partition") }}
```

Comment:
let's add the tests for both partition keys types ie string vs list

Reply:
Done